src/components/todoInput.js

```diff
@@ -0,0 +1,28 @@
+import { actionTypes } from '../utils/constants.js';
+import reducer from '../store/reducer.js';
+import createStore from '../store/store.js';
+import todoList from './todoList.js';
+import todoCount from './todoCount.js';
+
+
+const todoInput = () => {
+	const $input = document.getElementById('new-todo-title');
+	const store = createStore(reducer);
+
+		$input.addEventListener('keypress', e => {
+			if(e.key === 'Enter' && e.target.value.length > 0) {
+				store.dispatch({
+						type: actionTypes.ADD,
+						content: $input.value
+				})
+
+				todoList();
+				todoCount();
+
+				$input.value = '';
+			}
```

Comment:
```suggestion
		$input.addEventListener('keypress', { target, key } => {
			if(key === 'Enter' && target.value) {
				store.dispatch({
						type: actionTypes.ADD,
						content: $input.value
				})
				todoList();
				todoCount();
				$input.value = '';
			}
```
이 부분은 위와 같이 수정하면 간소화시키고 가독성을 높일 수 있을거 같습니다!